docs/tutorials/bigip-ha.rst

```diff
@@ -0,0 +1,130 @@
+.. index::
+   single: BIG-IP Controller
+   single: BIG-IP high availability
+   single: Cloud Foundry
+   single: Kubernetes
+   single: OpenShift
+   single: Marathon
+
+.. _manage BIG-IP HA:
+
+How to manage BIG-IP HA pairs
+=============================
+
+The F5 Container Connectors provide platform-native integrations for BIG-IP devices from PaaS providers like Cloud Foundry, Kubernetes, Mesos, & OpenShift. The BIG-IP Controllers for these platforms translate native commands to F5 Python SDK/iControl REST calls. [#cccl]_
+
+You can use any BIG-IP Controller to manage a BIG-IP HA active-standby pair or device group. While the platform details vary, all of the Controllers operate on the same basic principle: To provide redundancy, run one (1) Controller instance for each BIG-IP device.
+
+**For example**:
+
+You have one (1) active and one (1) standby BIG-IP device. You want to manage a Kubernetes Cluster in a single partition on the BIG-IP system. For your HA setup, you'd deploy two (2) |kctlr| instances - one for each BIG-IP device. To ensure Controller HA, deploy each Controller instance on a separate node in the cluster.
+
+.. figure:: /_static/media/bigip-ha.png
+   :alt: A diagram showing a BIG-IP active-standby device pair and 2 BIG-IP Controllers, running on separate nodes in a Kubernetes Cluster.
+
+.. [#cccl] See `Introduction to F5 Common Controller Core Library <https://devcentral.f5.com/articles/introduction-to-f5-common-controller-core-library-cccl-28355>`_ on DevCentral for more information.
+
+BIG-IP config sync
+------------------
+
+.. warning::
+
+   F5 does not recommended using automatic configuration sync for BIG-IP devices managed by BIG-IP Controllers. The BIG-IP Controller automatically reconfigures its managed device if it discovers changes from what it knows to be the desired state.
```

Comment:
Should we expand on what _desired state_ means? We should let the reader know that this state comes from the orchestration/cluster (e.g. virtual servers, backend pool members/endpoints).

Reply:
replaced with the standard warning used in the controller product docs (see http://clouddocs.f5.com/products/connectors/k8s-bigip-ctlr/v1.3/ for example).

Reply:
Perfect!